### ISSUE TITLE
Using additional headers

### ISSUE THREAD

**Issue by saskiaschild** [s-u/Rserve#177]
I couldn't find any information about using additional headers, e.g. authorization header. Is this even possible? In my case I would need the authorization header for sending API tokens.

**Comment by s-u** [comment]
@saskiaschild yes, (I assume you are talking about headers in the response to HTTP requests) the third entry in the result are the headers you wish to send:

                   list(payload[, content-type[, headers[, status code]]])
                   
                   payload: can be a character vector of length one or a
                   raw vector. if the character vector is named "file" then
                   the content of a file of that name is the payload
                   
                   content-type: must be a character vector of length one
                   or NULL (if present, else default is "text/html")
                   
                   headers: must be a character vector - the elements will
                   have CRLF appended and neither Content-type nor
                   Content-length may be used
                   
                   status code: must be an integer if present (default is 200)

**Comment by saskiaschild** [comment]
Thank you for the response! Sorry for not properly specifying my question, but I meant headers in the HTTP request. I'm using FastRWeb and Rserve and I tried to send the header "Authorization: Bearer <token>" in the request to Rserve/FastRWeb, but it was not found in the request. I tried to access it via `request$headers` but it was empty. Also any other header (besides, Content-Type, Content-Length and Host) was not included in the request. Is there a reason or am I accessing it wrong somehow?

Thanks in advance for your answer!
